### PR TITLE
changed the field to textarea

### LINKED ISSUE #5265: Comment input field should be a textarea
https://github.com/plone/volto/blob/c71a6ca108edca42dac2171e8c521cb26b92ce2a/src/components/theme/Comments/CommentEditModal.jsx#L62

### PR DESCRIPTION
**Pull request by Ravi-kumar9347**
Fixes #5265

### CODE REVIEW

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 in the future, read and follow Item 3 in [Things not to do](https://6.docs.plone.org/contributing/first-time.html#things-not-to-do), especially when you open multiple PRs from your same branch.

**Comment by Ravi-kumar9347** [comment]
@stevepiercy  i read that docs. but now what i have to do

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 it's a mystery to me why you deleted your branch `textarea`, closed your other PR https://github.com/plone/volto/pull/5572, then created a new PR from the same named branch. That is poor management of git branches and GitHub. Instead you should continue on your branch, even if it has mistakes in it, correcting the mistakes and pushing your commits.

Additionally you failed to follow [Create a pull request from your fork](https://6.docs.plone.org/contributing/first-time.html#create-a-pull-request-from-your-fork) items 2 and 3. Also because you closed a related PR, you should include in your description, "Supersedes #5572, MY-EXPLANATION-FOR-WHY-I-CLOSED-THE-OTHER-PR". Failing to do these things makes it difficult for core developers to track down what you previously did and to follow up on their previous review comments.

**Comment by Ravi-kumar9347** [comment]
@stevepiercy  sorry for my mistakes, as i am i begginer i made some mistakes and from now i dont repeat them. but can you please guide me what i have to do now.

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 I don't know how to make it any clearer than my previous comment. Sorry.

**Comment by Ravi-kumar9347** [comment]
@stevepiercy sorry for my inconvenience, by mistake i closed the before pr, but now onwards i dont repeat them. can i work now on this branch or not

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 what is not clear in the [second paragraph of my comment](https://github.com/plone/volto/pull/5579#issuecomment-1870169903)?

**Comment by Ravi-kumar9347** [comment]
@stevepiercy #5572 sorry for closing this pr. the reason is at that time i dont know how to handle the pr and some other branch realated things. now i have knowledge towards how to handle them and all other related things.
now can i work upon this pr? if yes, can you check my pr once. 
if any other things i need to change , can you guide me those.
thank you

File: packages/volto/news/5265.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Changed the field type from text to textarea in comment input field. @Ravi-kumar9347
```
Comment (davisagli) [inline packages/volto/news/5265.bugfix:1]:
```suggestion
Use a textarea in the form for editing an existing comment. @Ravi-kumar9347
```

**Review by davisagli — APPROVED** [review]
@Ravi-kumar9347 You missed the instruction from https://6.docs.plone.org/contributing/first-time.html#create-a-pull-request-from-your-fork to add "fixes #5265" in the PR description, so that it is linked to the issue it solves. I took care of that for you so we can get this merged, but please take care in future PRs. Thanks for the contribution.

**Comment by Ravi-kumar9347** [comment]
@davisagli sorry for my all mistakes and i will go through the docs and understand the all instructions. in future i dont repeat these type of mistakes.
thank you.

**Pull request merged**